block/batch.go

```diff
@@ -6,14 +6,16 @@ import (
 
 // BatchQueue is a queue of transaction batches with timestamps
 type BatchQueue struct {
-	queue []BatchWithTime
-	mu    sync.Mutex
+	queue    []BatchWithTime
+	mu       sync.Mutex
+	notifyCh chan struct{}
```

Comment:
introduced notification to batch for using in case of lazy aggregator instead of mempool txsAvailable